keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/probabilistic/CauchyVertexTest.java

```diff
@@ -109,7 +110,7 @@ public void dLogProbMatchesFiniteDifferenceCalculationFordPdlocation() {
 
         moveAlongDistributionAndTestGradientOnARangeOfHyperParameterValues(
             Nd4jDoubleTensor.scalar(1.0),
-            Nd4jDoubleTensor.scalar(1.5),
```

Comment:
Why the change?

Reply:
This was just to match the other test and run over the full range of the uniform.